tests/api_test.py

```diff
@@ -114,6 +130,55 @@ def test_create_newdle_duplicate_timeslot(flask_client, dummy_uid):
     }
 
 
+@pytest.mark.usefixtures('db_session')
+def test_create_newdle_participant_email_sending(flask_client, dummy_uid, mocker):
+    notif_func = mocker.patch('newdle.api.notify_newdle_participants')
```

Comment:
you can also use the `locmem` email backend; that way all emails are written to a list instead of being actually sent

Reply:
Oh cool!

Reply:
OK, I still don't know how to get the backend, since it seems like it's not persisted anywhere. Anyway, I am already using it to avoid methods that send e-mails sending. In this method, however, I'd like to count how many mails are sent.

Reply:
`from newdle.vendor import django_mail`

`django_mail.outbox`

Reply:
Thanks!